config.go

```diff
@@ -9,8 +9,9 @@ const (
 )
 
 type config struct {
-	StorageType string `envconfig:"STORAGE_ADAPTER" default:"memory"`
-	NumLines    int    `envconfig:"NUMBER_OF_LINES" default:"1000"`
+	StorageType    string `envconfig:"STORAGE_ADAPTER" default:"memory"`
+	NumLines       int    `envconfig:"NUMBER_OF_LINES" default:"1000"`
+	AggregatorType string `envconfig:"AGGREGATOR_TYPE" default:"nsq"`
```

Comment:
nevermind, I saw below that you have split configs apart and there's another in `log/config.go`, so you've already done this :+1: